### ISSUE TITLE
aria-labelledby pointing to children fails validation

### ISSUE THREAD

**Issue by nutbreaker** [validator/validator#2021]
### Spec URL

https://www.w3.org/TR/WCAG20-TECHS/ARIA13.html#ARIA13-examples

### Test case

```html
<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Books</title>

</head>

<body>
    <a href="book?id=13"
        aria-labelledby="card-figcaption-title-13 card-figcaption-subtitle-13 card-figcaption-unavailable-13 card-figcaption-info-13">
        <figure class="card">
            <img class="card-img" src="img_uploaded/943793e71aa60bc03195e31d1c17e47ebf15fad9.jpg"
                alt="The Subtle Art Of...">
            <figcaption class="card-figcaption">
                <h3 id="card-figcaption-title-13" class="card-figcaption-title">The Subtle Art Of...</h3>
                <p id="card-figcaption-subtitle-13" class="card-figcaption-subtitle">Mark Manson</p>
                <span id="card-figcaption-info-13" class="card-figcaption-info">Seller: Verogo33</span>
            </figcaption>
        </figure>
    </a>
</body>

</html>
```

<img width="1624" height="95" alt="Image" src="https://github.com/user-attachments/assets/69c1671e-09a1-40b0-8731-8e67f2bbb902" />

**Comment by nschonni** [comment]
The issue is you're using multiple IDs rather than a single value

**Comment by nutbreaker** [comment]
@nschonni wow, thank you for your quick answer!

I just tested it with a single ID and can confirm that it passes validation. However, the [ARIA13-description](https://www.w3.org/TR/WCAG20-TECHS/ARIA13.html#ARIA13-description) section seems to mention that `aria-describedby` accepts multiple IDs.

**Comment by pkra** [comment]
@nutbreaker the ID `card-figcaption-unavailable-13` is the problem. It's not in the document but one of the IDREFs in the aria-labelledby. If you remove it, the example will validate.

**Comment by sideshowbarker** [comment]
Closing per comments, as it seems that the checker is behaving as expected here, and conforming

**Comment by pkra** [comment]
@sideshowbarker looking at it again, perhaps the error message could be improved? I can see how one might interpret it to say that only one idref is allowed. (I suppose it would be even nicer if the checker spits out the "bad" idref(s).)

**Comment by sideshowbarker** [comment]
> [@sideshowbarker](https://github.com/sideshowbarker) looking at it again, perhaps the error message could be improved? I can see how one might interpret it to say that only one idref is allowed. (I suppose it would be even nicer if the checker spits out the "bad" idref(s).)

Yeah, you’re right. I will take a look at trying to improve it.

**Comment by sideshowbarker** [comment]
@nutbreaker Thanks for taking time to report this. @pkra Thanks for suggesting how we can make things more clear.

Because of that, for the case reported in the issue description — as well as for the cases of some related ARIA attributes — we now have the checker reporting a (hopefully) much-more-helpful error message:

> _Error: The `aria-labelledby` attribute references `card-figcaption-unavailable-13`, which is not the ID of any element in this document._

I’ve implemented that now, and deployed it:

https://validator.w3.org/nu/?showsource=yes&doc=data%3Atext%2Fhtml%3Bcharset%3Dutf-8%3Bbase64%2CPCFET0NUWVBFIGh0bWw%2BDQo8aHRtbCBsYW5nPSJlbiI%2BDQoNCjxoZWFkPg0KICAgIDxtZXRhIGNoYXJzZXQ9IlVURi04Ij4NCiAgICA8bWV0YSBuYW1lPSJ2aWV3cG9ydCIgY29udGVudD0id2lkdGg9ZGV2aWNlLXdpZHRoLCBpbml0aWFsLXNjYWxlPTEuMCI%2BDQogICAgPHRpdGxlPkJvb2tzPC90aXRsZT4NCg0KPC9oZWFkPg0KDQo8Ym9keT4NCiAgICA8YSBocmVmPSJib29rP2lkPTEzIg0KICAgICAgICBhcmlhLWxhYmVsbGVkYnk9ImNhcmQtZmlnY2FwdGlvbi10aXRsZS0xMyBjYXJkLWZpZ2NhcHRpb24tc3VidGl0bGUtMTMgY2FyZC1maWdjYXB0aW9uLXVuYXZhaWxhYmxlLTEzIGNhcmQtZmlnY2FwdGlvbi1pbmZvLTEzIj4NCiAgICAgICAgPGZpZ3VyZSBjbGFzcz0iY2FyZCI%2BDQogICAgICAgICAgICA8aW1nIGNsYXNzPSJjYXJkLWltZyIgc3JjPSJpbWdfdXBsb2FkZWQvOTQzNzkzZTcxYWE2MGJjMDMxOTVlMzFkMWMxN2U0N2ViZjE1ZmFkOS5qcGciDQogICAgICAgICAgICAgICAgYWx0PSJUaGUgU3VidGxlIEFydCBPZi4uLiI%2BDQogICAgICAgICAgICA8ZmlnY2FwdGlvbiBjbGFzcz0iY2FyZC1maWdjYXB0aW9uIj4NCiAgICAgICAgICAgICAgICA8aDMgaWQ9ImNhcmQtZmlnY2FwdGlvbi10aXRsZS0xMyIgY2xhc3M9ImNhcmQtZmlnY2FwdGlvbi10aXRsZSI%2BVGhlIFN1YnRsZSBBcnQgT2YuLi48L2gzPg0KICAgICAgICAgICAgICAgIDxwIGlkPSJjYXJkLWZpZ2NhcHRpb24tc3VidGl0bGUtMTMiIGNsYXNzPSJjYXJkLWZpZ2NhcHRpb24tc3VidGl0bGUiPk1hcmsgTWFuc29uPC9wPg0KICAgICAgICAgICAgICAgIDxzcGFuIGlkPSJjYXJkLWZpZ2NhcHRpb24taW5mby0xMyIgY2xhc3M9ImNhcmQtZmlnY2FwdGlvbi1pbmZvIj5TZWxsZXI6IFZlcm9nbzMzPC9zcGFuPg0KICAgICAgICAgICAgPC9maWdjYXB0aW9uPg0KICAgICAgICA8L2ZpZ3VyZT4NCiAgICA8L2E%2BDQo8L2JvZHk%2BDQoNCjwvaHRtbD4%3D

**Comment by nutbreaker** [comment]
@sideshowbarker purefection thank you verry much.
